examples/lexer/python_indentation/python_indentation.js

```diff
@@ -70,13 +70,26 @@ function matchIndenBase(text, matchedTokens, groups, type) {
         }
 
         if (currIndentLevel !== undefined) {
-            let lastIndentLEvel = _.last(identStack)
-            if (currIndentLevel > lastIndentLEvel && type === "indent") {
-                identStack.push(currIndentLevel)
+            let lastIndentLevel = _.last(indentStack)
+            if (currIndentLevel > lastIndentLevel && type === "indent") {
+                indentStack.push(currIndentLevel)
                 return match
             }
-            else if (currIndentLevel < lastIndentLEvel && type === "outdent") {
-                identStack.pop()
+            else if (currIndentLevel < lastIndentLevel && type === "outdent") {
+                //if we need more than one outdent token, add all but the last one
+                if(indentStack.length > 2){
+                  const image = "";
+                  const offset = chevrotain.getEndOffset(_.last(matchedTokens)) + 1
+                  const line = chevrotain.getEndLine(_.last(matchedTokens))
+                  const column = chevrotain.getEndColumn(_.last(matchedTokens)) + 1
+                  while(indentStack.length > 2 &&
```

Comment:
I may change this loop in a separate commit, basically the condition is quite complex.
I think it would be easier to understand if first the **number** of additional required Outdents is computed.
And only then will they be added in a **for** loop.